src/peasy/PeasyCam.java

```diff
@@ -503,6 +503,21 @@ public void pan(final double dx, final double dy) {
 		feed();
 	}
 
+	public void move(final double dx, final double dy, final double dz) {
+		center = center.add(rotation.applyTo(new Vector3D(dx, dy, dz)));
+		feed();
+	}
+
+	public void forward(final double distance) {
```

Comment:
Nice! These should just call `move(0, 0, -distance);` and `move(0, 0, distance);`.

Reply:
Should be resolved in f2d1b9e, see comment for more info !